egs2/seame/asr1/conf/decode_asr.yaml

```diff
@@ -0,0 +1,3 @@
+lm_weight: 0.2
+ctc_weight: 0.4
+beam_size: 30
```

Comment:
Is it after tuning?
Beam size is a bit larger than usual (20) (it’s OK if this is after some tuning)

Reply:
Yes, the weights and beam size are tuned. I can make the beam size 10 (only 0.1% absolute WER increase in both eval sets).

Reply:
ok!
either is fine!